test/runtests.jl

```diff
@@ -104,7 +104,24 @@ using ForwardDiff
 
         @test chain1[1].params == val
     end
-    
+
+    @testset "is_symmetric_proposal" begin
+        # Model definition.
+        m1 = DensityModel(s -> logpdf(Normal(), s.x) + logpdf(Normal(5,.7), s.y))
+
+        # Set up the proposal.
+        p1 = (x=RandomWalkProposal(Normal(0,.5)), y=RandomWalkProposal(Normal(0,.5)))
+        AdvancedMH.is_symmetric_proposal(proposal::typeof(p1)) = true
```

Comment:
This is not a very "nice" example - usually one would want to define the trait for a custom proposal or distribution.

Reply:
Maybe define a custom distribution, e.g., of normal distributions with zero mean?

Reply:
Sounds good to me.

Reply:
Just modified the test.